repository.yaml

```diff
@@ -11,6 +11,9 @@ templates:
   - name: java11
     lang: java
     path: /templates/java11
+  - name: csharp
+    lang: csharp
+    path: /templates/dotnet
```

Comment:
The template name and directory should be the same. I'd suggest csharp for both, since although it's using the dotnet SDK, it's a C# template.